src/components/BaseCard.vue

```diff
@@ -67,7 +69,7 @@ export default {
   components: { ReadingTime, BaseTag },
   props: {
     post: {
-      type: Object,
+      type: [Object, String],
       default: () => {},
```

Comment:
The default value will be `undefined` in this case
We should either convert to 
```suggestion
      default: () => ({}),
```
or 
```suggestion
      default: () => { return {} },
```
If we want an empty object as a default value